src/IIIFPresentation/Models/API/Collection/HierarchicalCollection.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Models.API.Collection;
+
+public class HierarchicalCollection
+{
+    [JsonPropertyName("@context")]
+    public required string Context { get; set; }
+    
+    public required string Id { get; set; }
+    
+    public PresentationType Type { get; set; }
+    
+    public Dictionary<string, List<string>>? Label { get; set; }
```

Comment:
`LanguageMap` like `FlatCollection`? (same for `Item`, `PartOf`, `SeeAlso` etc?)